src/devices/Dhtxx/DhtSensor.cs

```diff
@@ -0,0 +1,218 @@
+
+using System;
+using System.Device.Gpio;
+using System.Diagnostics;
+
+namespace Iot.Device.DHTxx
+{
+    public class DelayMS
+    {
+
```

Comment:
feedback for all similar places: extra enter (see other files for style or https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)